src/player.js

```diff
@@ -2601,6 +2605,20 @@ export default class Player extends FakeEventTarget {
     }
   }
 
+  /**
+   * Sets the video track selected by the user.
+   * @returns {void}
+   * @private
+   */
+  _setDefaultVideoTrack() {
+    const selectedVideoTrack = this._getVideoTracks()
```

Comment:
is this for ABR or only manual selected bitrate?
If user selected the video track then this will be OK, but if this happens on every loading of media it means this will apply for ABR mode as well.
For ABR mode we should use the estimated bitrate(which for HLS doesn't work well)

Reply:
it's only for manual selection. on ABR mode `this._playbackAttributesState.videoHeight` is undefined

Reply:
I think the code is a bit too implicit in the way I read it, as it relies on the fact we have undefined value when we call `enableAdaptiveBitrate`.
Maybe better to put it in an if statement that relates on `isAdaptiveBitrateEnabled`.
```javascript
if (this.isAdaptiveBitrateEnabled()) {
  /*this is sort of like auto option for audio and text track*/
} else {
  /*handle manual selection*/
}
```

Reply:
`isAdaptiveBitrateEnabled` will be true after change media. this is the default.